controllers/admin/AdminImportController.php

```diff
@@ -4691,15 +4725,13 @@ public function ajaxProcessImport()
         die(json_encode($results));
     }
 
-    public function initModal()
+    /**
+     * Gets session from symfony container.
+     *
+     * @return \Symfony\Component\HttpFoundation\Session\Session
+     */
+    private function getSession()
     {
-        parent::initModal();
-        $modal_content = $this->context->smarty->fetch('controllers/import/modal_import_progress.tpl');
-        $this->modals[] = array(
-             'modal_id' => 'importProgress',
-             'modal_class' => 'modal-md',
-             'modal_title' => $this->trans('Importing your data...', array(), 'Admin.Advparameters.Notification'),
-             'modal_content' => $modal_content,
-         );
+        return \PrestaShop\PrestaShop\Adapter\SymfonyContainer::getInstance()->get('session');
```

Comment:
yes, it will be removed with next PR when import logic will be migrated to adapters, but now to make this PR fully functional I had to add some code to the legacy controller.